src/test/java/nextstep/courses/domain/ImageCapacityTest.java

```diff
@@ -0,0 +1,55 @@
+package nextstep.courses.domain;
+
+import nextstep.courses.domain.SessionImage.ImageCapacity;
+import nextstep.courses.domain.SessionImage.ImageSize;
+import nextstep.courses.domain.SessionImage.ImageType;
+import nextstep.courses.domain.SessionImage.SessionImage;
+import nextstep.payments.domain.Payment;
+import nextstep.users.domain.NsUser;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+import org.junit.jupiter.params.provider.ValueSource;
+
+import java.time.LocalDateTime;
+
+import static org.assertj.core.api.Assertions.assertThatIllegalArgumentException;
+import static org.junit.jupiter.api.Assertions.assertDoesNotThrow;
+
+public class ImageCapacityTest {
+
+    public static final int IMAGE_CAPACITY = 1;
+    public static final int WIDTH = 300;
+    public static final int HEIGHT = 200;
+    public static final String JPG = "jpg";
+    ImageCapacity imageCapacity;
+    ImageSize imageSize;
+    ImageType imageType;
+    SessionImage sessionImage;
+    @BeforeEach
+    public void setUp() {
+
+        imageCapacity = new ImageCapacity(IMAGE_CAPACITY);
+        imageSize = new ImageSize(WIDTH, HEIGHT);
+        imageType = ImageType.jpg;
+
+    }
+
+
+    @Test
+    public void 이미지크기는_1MB이하_테스트() {
+        assertDoesNotThrow(() -> {
+            sessionImage = new SessionImage(imageCapacity,imageType,imageSize);
```

Comment:
```suggestion
            new SessionImage(imageCapacity, imageType, imageSize);
```
생성 검증이므로 변수 할당까지는 할 필요는 없어보여요

Reply:
디테일한 부분 체크해주셨네요 감사합니다! 생성검증 관련 부분은 불필요한 변수할당은 안해봐야겠습니다!